sensor_msgs/PointCloud2 cloud
std_msgs/Int64 size_left
